projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -74,10 +74,6 @@ public function reregister_menu_items() {
 		$this->add_my_home_menu();
 		$this->remove_gutenberg_menu();
 
-		if ( ! get_option( 'wpcom_is_staging_site' ) ) {
```

Comment:
We still need to keep it for the default style 🥲

Reply:
Yes, we don't want to change anything in the Default style. I suggest to use the same code we are using in other PRs:
```
if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
```